server/docs/Recommendation System.md

```
@@ -0,0 +1,523 @@
+#Recommendations (context-driven, system-initiated)
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)